modules/lib/src/main/resources/i18n/phrases.properties

```diff
@@ -235,6 +235,7 @@ field.rich.text = Rich Text Editor
 field.rich.text.help = Write text and embed images
 field.fragment=Fragment
 field.fragment.help=Reusable components
+field.image.help = Upload or use existing image
```

Comment:
We also need to rename the key. The way it is now is too generic, and it doesn't have anything to do with field either.

We have a separate section of keys that start with "tooltip", so I suggest to add the new key there and call it "`tooltip.button.uploadImage`".